pexpect/__init__.py

```diff
@@ -68,7 +68,12 @@
 
 from .exceptions import ExceptionPexpect, EOF, TIMEOUT
 from .utils import split_command_line, which, is_executable_file
-from .pty_spawn import spawn, spawnu, PY3
+
+if sys.platform == 'win32':
+    from .popen_spawn import (
+        PopenSpawn as spawn, PopenSpawnUnicode as spawnu, PY3)
+else:
+    from .pty_spawn import spawn, spawnu, PY3
```

Comment:
I don't think this is a good idea - it would mean two different interfaces would be presented as the same thing, which is bound to cause bugs. I'd rather leave it up to users to do the conditional import if that's OK for them.